commands/user.go

```diff
@@ -402,3 +427,84 @@ func listUsersCmdF(c client.Client, command *cobra.Command, args []string) error
 
 	return nil
 }
+
+func generateTokenForAUserCmdF(c client.Client, command *cobra.Command, args []string) error {
+	if len(args) != 2 {
+		return errors.New("expected at least two arguments. See help text for details")
+	}
+
+	userArg := args[0]
+	user := getUserFromUserArg(c, userArg)
+	if user == nil {
+		return errors.Errorf("could not retrieve user information of %q", userArg)
+	}
+
+	token, res := c.CreateUserAccessToken(user.Id, args[1])
+	if res.Error != nil {
+		return errors.Errorf("could not create token for %q: %s", userArg, res.Error.Error())
+	}
+	printer.Print(fmt.Sprintf("%s : %s", token.Token, token.Description))
+
+	return nil
+}
+
+func listTokensOfAUserCmdF(c client.Client, command *cobra.Command, args []string) error {
+	if len(args) != 1 {
+		return errors.New("expected at least one argument. See help text for details")
+	}
```

Comment:
We can replace this check by adding `Args: cobra.ExactArgs(1)` in the command struct

Reply:
Added. 👍